src/avt/Pipeline/CMakeLists.txt

```diff
@@ -264,8 +270,11 @@ IF(VISIT_PARALLEL)
                           vtkCommonDataModel vtkCommonMath vtkCommonTransforms
                           vtkFiltersGeometry
                           ${ZLIB_LIBRARY})
-    if(VTKH_FOUND)
-        target_link_libraries(avtpipeline_par vtkh)
+    if(HAVE_LIBVTKM)
+        target_link_libraries(avtpipeline_ser vtkm_cont vtkm_filter_contour vtkm_filter_entity_extraction)
+        if(VTKm_ENABLE_KOKKOS)
+            target_link_libraries(avtpipeline_ser amdhip64)
+        endif()
```

Comment:
Note: We are missing link of vtk-m in for avtpipeline_par.

I will add this in a follow up PR.

Reply:
PR w/ Fix for 3.3RC
https://github.com/visit-dav/visit/pull/18542/